modules/fscloud/main.tf

```diff
@@ -52,6 +52,9 @@ locals {
       object_versioning             = config.object_versioning
       retention_rule                = config.retention_rule
       cbr_rules                     = config.cbr_rules
+      single_site_location          = config.single_site_location
```

Comment:
I don't think single site location will pass fscloud requirements. However I don't think you should remove it from the variable objects because we said we would keep this input variable the same as the one in the buckets submodule, so perhaps add variable validation to ensure someone doesnt set this. Same way we did for `kms_encryption_enabled` here -> https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/c8477a8cc277c200c1bb0dbfa2e55a312d1768e4/modules/fscloud/main.tf#L9C45-L9C67

Reply:
Also just double check what fscloud says about single site COS buckets first

Reply:
it looks it supports all 3 https://cloud.ibm.com/docs/framework-financial-services?topic=framework-financial-services-vpc-architecture-storage